editor/src/diagram/di.config.ts

```diff
@@ -51,81 +64,105 @@ const ivyDiagramModule = new ContainerModule((bind, unbind, isBound, rebind) =>
 
   const context = { bind, unbind, isBound, rebind };
 
-  configureModelElement(context, DefaultTypes.GRAPH, IvyGLSPGraph, SGraphView);
-
-  configureModelElement(context, EventTypes.START, StartEventNode, EventNodeView);
-  configureModelElement(context, EventTypes.START_ERROR, StartEventNode, EventNodeView);
-  configureModelElement(context, EventTypes.START_SIGNAL, StartEventNode, EventNodeView);
-  configureModelElement(context, EventTypes.START_PROGRAM, StartEventNode, EventNodeView, { enable: [editSourceFeature] });
-  configureModelElement(context, EventTypes.START_SUB, StartEventNode, EventNodeView);
-  configureModelElement(context, EventTypes.START_WS, StartEventNode, EventNodeView);
-  configureModelElement(context, EventTypes.START_HD, StartEventNode, EventNodeView);
-  configureModelElement(context, EventTypes.START_HD_METHOD, StartEventNode, EventNodeView);
-  configureModelElement(context, EventTypes.START_HD_EVENT, StartEventNode, EventNodeView);
-  configureModelElement(context, EventTypes.START_EMBEDDED, StartEventNode, EventNodeView);
-  configureModelElement(context, EventTypes.END, EndEventNode, EventNodeView);
-  configureModelElement(context, EventTypes.END_ERROR, EndEventNode, EventNodeView);
-  configureModelElement(context, EventTypes.END_PAGE, EndEventNode, EventNodeView);
-  configureModelElement(context, EventTypes.END_SUB, EndEventNode, EventNodeView);
-  configureModelElement(context, EventTypes.END_WS, EndEventNode, EventNodeView);
-  configureModelElement(context, EventTypes.END_HD, EndEventNode, EventNodeView);
-  configureModelElement(context, EventTypes.END_HD_EXIT, EndEventNode, EventNodeView);
-  configureModelElement(context, EventTypes.END_EMBEDDED, StartEventNode, EventNodeView);
-  configureModelElement(context, EventTypes.INTERMEDIATE, EventNode, IntermediateEventNodeView);
```

Comment:
@ivy-lli Do I also need to register the default intermediate node here? In the end, there are only two intermediate events on the palette (wait and task).